### ISSUE TITLE
[Feature Request] Test mode

### ISSUE THREAD

**Issue by TheDMV2** [Cloud9Developer/Jellyfin-Newsletter-Plugin#61]
I'd love to be able to send a sample newsletter.
And have it go to a separate, single email address, not part of the normal list, where you can hit "test" and it will send what the newsletter would be, if it were sent now (without affecting/resetting the db). This way we can make changes, see results and debug more quickly.

This is more than just "check the mail server is working", but actually sends the full newsletter.

**Comment by Cloud9Developer** [comment]
This is actually already in my backlog, though slightly different (see #51 )

**Comment by TheDMV2** [comment]
While I wholeheartedly agree that a preview window would be fantastic, this is needed for further testing / debugging / designing.

Some mail servers rewrite and mangle content. Gmail is a good example of this.

Different mail clients have different levels of CSS compatibility. Seeing it in your browser isn't assurance that it will look the same to the client. Positioning of elements is a HUGE part of it. That's why we're still using nested tables in 2023, for example. SVG images is another example.

**Comment by Sammeeeeeeee** [comment]
I agree that we need sample emails. I have been trying for the past 30 minutes to rejiggle and rename stuff to get the scraper to notice a change so I can recive a test email to view my new HTML, but no luck 😤
